app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+  has_many :likes
+
+  def load_recent_three_posts
+    Post.last(3)
```

Comment:
- [ ] the user post should have the order with created_at.

Reply:
Ok thank you will do that